assets/styles/css/nullstyle.scss

```diff
@@ -0,0 +1,90 @@
+/* Указываем box sizing */
+
```

Comment:
Look, I think if you're doing some null styling, you should have a good understanding of css and understand why you're doing some null styling. It doesn't make any sense otherwise, you're just doing null styles to do null styles.